fdbserver/DDShardTracker.actor.cpp

```diff
@@ -1112,7 +1116,7 @@ ACTOR Future<Void> shardEvaluator(DataDistributionTracker* self,
 		onChange = onChange || shardMerger(self, keys, shardSize);
 	}
 	if (shouldSplit) {
-		RelocateReason reason = writeSplit ? RelocateReason::WRITE_SPLIT : RelocateReason::SIZE_SPLIT;
+		RelocateReason reason = sizeSplit ? RelocateReason::SIZE_SPLIT : RelocateReason::WRITE_SPLIT;
```

Comment:
Yeah. I change it on purpose. The motivation is `SIZE_SPLIT` is more stable than `WRITE_SPLIT`, so there are more chances to prevent a write split when we want to maintain the size split.
When a shard can be eligible for both `SIZE_SPLIT` and `WRITE_SPLIT`, even though the shard is not write-hot anymore after a while, we may still want to split it because the size is too large.